Upstream: yes

From faf6919f36eae556f1d4e40b8847070221f553c6 Mon Sep 17 00:00:00 2001
From: Paul Moore <paul@paul-moore.com>
Date: Wed, 10 Feb 2016 11:36:42 -0500
Subject: [PATCH] system: use sys/prctl.h instead of linux/prctl.h

This should help on systems that use alternate libc implementations.

Reported-by: anarchpenguin
Reported-by: Kylie McClain somasis@exherbo.org
Signed-off-by: Paul Moore <paul@paul-moore.com>
---
 src/system.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/system.h b/src/system.h
index 4660679..02fd1e9 100644
--- a/src/system.h
+++ b/src/system.h
@@ -23,7 +23,7 @@
 #define _SYSTEM_H
 
 #include <linux/filter.h>
-#include <linux/prctl.h>
+#include <sys/prctl.h>
 
 #include "configure.h"
 
